tools/vscode/refresh_compdb.sh

```diff
@@ -2,11 +2,16 @@
 
 [[ -z "${SKIP_PROTO_FORMAT}" ]] && tools/proto_format/proto_format.sh fix
 
+source_dirs=("//source/..." "//test/..." "//tools/...")
+if [ "$1" == "--include-contrib" ]; then
```

Comment:
lets add this in `gen_compilation_database.py` instead of here.

Reply:
Moved to gen_compilation_database.py.  PTAL.